common/src/test/java/uk/gov/justice/services/common/converter/JsonObjectToObjectConverterTest.java

```diff
@@ -98,7 +117,7 @@ private JsonObject jsonObject() {
         private final InternalPojo internalPojo;
 
 
-        public Pojo(UUID id, String name, List<String> attributes, InternalPojo internalPojo) {
+        public Pojo(@JsonProperty("id") UUID id, @JsonProperty("name") String name, @JsonProperty("attributes") List<String> attributes, @JsonProperty("internalPojo") InternalPojo internalPojo) {
```

Comment:
Do we need these? The objectMapper should be able to get the names from the JDK8 parameter names.